lib/taskana-core/src/main/java/pro/taskana/user/internal/UserMapperSqlProvider.java

```diff
@@ -65,6 +72,26 @@ public static String insertGroupsOracle() {
         + CLOSING_SCRIPT_TAG;
   }
 
+  public static String insertPermissions() {
```

Comment:
Where is this method used in the mapper?

Reply:
should be used via the Annotation [here](https://github.com/nyuuyn/taskana/blob/f885c82fb3d8a59971b34ee0719edace2e1b06e9/lib/taskana-core/src/main/java/pro/taskana/user/internal/UserMapper.java#L74)